### PR TITLE
Improve File Handling in Kraken CSV

### PR DESCRIPTION
**Pull request by macanudo527**
This fixes some alternative market handling in the CCXT pair converter as well as allow the pair converter to continue with REST price resolution if the file isn't found on Google Drive like what is described in #112 

EDIT: Let me do a full pull and double-check if all alternative markets are added. That could be ~12 hours.

### CODE REVIEW

**Comment by macanudo527** [comment]
I spoke too soon. I was only missing two other markets. We will most likely have to work out a better solution to just manually adding these though.

**Comment by eprbell** [comment]
Sounds good, thanks for jumping on this!

**Comment by macanudo527** [comment]
This still results in the program self-terminating when pulling pricing for 6+ different crypto assets on my machine with 40gb a RAM. I will open another PR soon to better handle CSV caching.

**Comment by macanudo527** [comment]
Do you want to merge this PR for now, and then I can open another that splits up the CSVs since that will be a pretty big revision?

**Comment by eprbell** [comment]
> Do you want to merge this PR for now, and then I can open another that splits up the CSVs since that will be a pretty big revision?

Ok, will review in the next couple of days.

File: src/dali/plugin/pair_converter/ccxt.py [hunk 1]
```diff
@@ -240,8 +248,8 @@ def get_historic_bar_from_native_source(self, timestamp: datetime, from_asset: s
                 self.__logger.debug("Using default exchange %s type for %s.", self.__default_exchange, exchange)
             exchange = self.__default_exchange
 
-        # Caching of exchanges
-        if exchange not in self.__exchanges:
+        # The exchange could have been added as an alt if so markets wouldn't have been built
```
Comment (eprbell) [inline src/dali/plugin/pair_converter/ccxt.py:251]: as an alt: if so

**Review by eprbell — CHANGES_REQUESTED** [review]
LGTM: just a nit in a comment.

**Comment by macanudo527** [comment]
This should be okay now. I have the chunking PR ready, just want to run it in the real-world and see if it holds up.

**Review by eprbell — APPROVED** [review]
LGTM

**Pull request merged**